cartoframes/credentials.py

```diff
@@ -1,4 +1,5 @@
 """Credentials management for cartoframes usage."""
+import appdirs
```

Comment:
I thought the normal pattern was to put Python standard libraries first, third party packages next, and then local modules, etc.

Reply:
:thinking: I think this is another change by autopep8. Do you want me to change it?